internal/cli/kraft/cloud/volume/import/import.go

```diff
@@ -0,0 +1,269 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2024, Unikraft GmbH and The KraftKit Authors.
+// Licensed under the BSD-3-Clause License (the "License").
+// You may not use this file except in compliance with the License.
+
+package vimport // "v(olume)import"; "import" is a reserved keyword
+
+import (
+	"context"
+	"crypto/tls"
+	"errors"
+	"fmt"
+	"io"
+	"net"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/spf13/cobra"
+
+	kraftcloud "sdk.kraft.cloud"
+	kcinstances "sdk.kraft.cloud/instances"
+	kcservices "sdk.kraft.cloud/services"
+	kcvolumes "sdk.kraft.cloud/volumes"
+
+	"kraftkit.sh/cmdfactory"
+	"kraftkit.sh/config"
+	"kraftkit.sh/initrd"
+	"kraftkit.sh/internal/cli/kraft/cloud/utils"
+	"kraftkit.sh/log"
+)
+
+type Options struct {
```

Comment:
```suggestion
type ImportOptions struct {
```

Reply:
All CLI programs have their options named `<Command>Options`.  Let's keep it this way.